README.md

```diff
@@ -2,24 +2,6 @@
 
 requires node >= 8
```

Comment:
is it still this version of node? I had nodejs 14 installed locally but typedoc convert had failed for me with `Unexpected token '??='`, could you please check the min version again?

Reply:
Seems new version of `typedoc` needs "node": ">= 16", I will change requirements